src/zero/api/handlers.clj

```diff
@@ -55,7 +55,7 @@
         (if-let [jwt (some-> request :oauth2/access-tokens :google :id-token
                              decode-jwt valid?)]
           (handler (assoc request :jwt jwt))
-          (response/redirect landing-uri))))))
+          (response/status (response/response "") 401))))))
```

Comment:
Seems a shame to waste that `""`.

Reply:
And maybe add an `Authenticate` header?

Reply:
https://tools.ietf.org/html/rfc7235#section-4.1
… though maybe Ring does that for us?

Reply:
I'm not sure...I'll take a look!

Reply:
It's not added by Ring, so I added it in :octocat: Lmk what you think!